public_website/templates/public_website/search.html

```diff
@@ -97,17 +102,24 @@
         {% if questions or articles %}
         <div class="head-area">
             <span class="results-count">
-                Showing {% if result_size == 1 %}1{% else %}{{ start_index}} - {{ end_index }} {% endif %} of {{ result_size }} result{{ result_size|pluralize }}
+                Showing {% if result_size == 1 %}1{% else %}{{ results.start_index}}-{{ results.end_index }}{% endif %} of {{ result_size }} result{{ result_size|pluralize }}
```

Comment:
unwanted change 1

Reply:
This issue is now fixed by #397.

There are two other issues remaining. 1. Order of questions on the public site and other dashboard search pages is affected by this PR (the rule is not obvious) and 2. questions with 2 answers are not sorted as expected (i.e. by the lower comment count).

Reply:
One more issue - the dashboard count of review answer is not updating. It is static at 112.

Reply:
1. Order of questions is fixed by #408.
2. To be further tested. (At present submitting a review is giving server error 500.)